client/src/styles/post-card.css

```diff
@@ -0,0 +1,189 @@
+@import url('https://fonts.googleapis.com/css2?family=Readex+Pro');
+.card{
+    width: 582px;
+    font-family: 'Readex Pro';
```

Comment:
I think we should save this font as a variable in file called constants maybe with our CSS constants like primary color and font and secondary

Reply:
should I use [config provider] (https://ant.design/components/config-provider)?

Reply:
https://www.w3schools.com/css/css3_variables.asp

Example: 
```css
:root {
  --main-color: #e8505b;
  --main-white: #ffffff;
  --main-black: #000000;
  --content-grey: #636d73;
  --content-blue-green: #3a5a6e;
  --border-white: #fafbfd;
  --pagination-color: #343338;
  --main-font: 'Open Sans', sans-serif;
  --box-shadow: 0px 1px 8px #142e6e1a;
  --card-shadow: 0px 3px 6px #00000014;
  --input-border: #a2c0d4;
}

body {
  font-family: var(--main-font);
}
```